javascripts/discourse/initializers/disco-toc-composer.js

```diff
@@ -0,0 +1,46 @@
+import { withPluginApi } from "discourse/lib/plugin-api";
+
+export default {
+  name: "disco-toc-composer",
+
+  initialize() {
+    withPluginApi("1.0.0", (api) => {
+      const currentUser = api.getCurrentUser();
+      const currentUserTL = currentUser ? currentUser.trust_level : "";
```

Comment:
It's a bit weird to default no user to an empty string. Maybe the code should `return` if there is no `currentUser`

Reply:
Nice catch, no need to do anything here for anons, indeed. Fixed!